docs/tutorial/creating-a-new-hardhat-project.md

```diff
@@ -79,7 +79,19 @@ require("@nomiclabs/hardhat-waffle");
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
-  solidity: "0.7.3",
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.4",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+        },
+      },
+    ],
+  },
```

Comment:
Why not just this?

```suggestion
  solidity: "0.8.4",
```

Enabling the compiler during development can make some things work worse

Reply:
"Enabling the compiler during development can make some things work worse" - Good to know, sounds like a point worth explaining particularly to newcomers

To be honest, I was following the tutorial, it failed to build and I just copied in something that works for me from another project :) 

Assuming the people have in mind for this tutorial is a complete newcomer they might really struggle with tripping up at the first hurdle when following instructions :) 

Cheers